weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -145,6 +145,10 @@ class NotFoundError(Exception):
 ObjRefListType = list[ri.InternalObjectRef]
 
 
+CLICKHOUSE_SINGLE_ROW_INSERT_BYTES_LIMIT = 3.5 * 1024 * 1024  # 3.5 MiB
+ENTITY_TOO_LARGE_PAYLOAD = '{"_weave": "<EXCEEDS_LIMITS>"}'
```

Comment:
Again, i would nest the `"<EXCEEDS_LIMITS>"`. Else we are sort of stuck with a string value here

Reply:
oh sorry, I understand.